include/deal.II/lac/matrix_block.h

```diff
@@ -405,8 +405,6 @@ class MatrixBlockVector : private AnyData
    */
   using AnyData::name;
   using AnyData::size;
-  using AnyData::subscribe;
-  using AnyData::unsubscribe;
```

Comment:
Don't ask me why this class imports these symbols. That whole `MatrixBlock` thing needs to be deprecated.